topic/machine-learning/llm-langchain/README.md

```diff
@@ -68,6 +68,16 @@ and [CrateDB].
   corresponding examples, [vector_search.py](vector_search.py),
   [document_loader.py](document_loader.py), and
   [conversational_memory.py](conversational_memory.py).
+  
+- `cratedb_rag_customer_support_langchain.ipynb` [![Open on GitHub](https://img.shields.io/badge/Open%20on-GitHub-lightgray?logo=GitHub)](cratedb_rag_customer_support_langchain.ipynb)[![Open in Colab](https://colab.research.google.com/github/crate/cratedb-examples/blob/marija%2Frag-fosdem/topic/machine-learning/llm-langchain/cratedb_rag_customer_support_langchain.ipynb)
```

Comment:
This link needs to target the main branch to make it work after being merged.

```suggestion
- `cratedb_rag_customer_support_langchain.ipynb` [![Open on GitHub](https://img.shields.io/badge/Open%20on-GitHub-lightgray?logo=GitHub)](cratedb_rag_customer_support_langchain.ipynb)[![Open in Colab](https://colab.research.google.com/github/crate/cratedb-examples/blob/main/topic/machine-learning/llm-langchain/cratedb_rag_customer_support_langchain.ipynb)
```

Reply:
👍